content/en/docs/components/trainer/user-guides/builtin-trainer/torchtune.md

```diff
@@ -0,0 +1,220 @@
+
+The supported model list can be seen in [this directory](https://github.com/kubeflow/trainer/tree/master/manifests/base/runtimes/torchtune). It's worth noticing that we do not support multi-node fine-tuning for now. Only one Pod will be launched for TorchTune LLM Trainer.
+
+In this page, we'll show a simple example of fine-tuning LLMs with TorchTune LLM Trainer in Kubeflow SDK. If you want to learn more about TorchTune LLM Trainer, please refer to [KEP-2401](https://github.com/kubeflow/trainer/tree/master/docs/proposals/2401-llm-trainer-v2) in Kubeflow Trainer.
+
+## Prerequisites
+
+Before exploring this guide, make sure to follow [the Getting Started guide](https://www.kubeflow.org/docs/components/trainer/getting-started/) to understand the basics of Kubeflow Trainer.
+
+```Python
+# Get torchtune-llama3.2-1b Kubeflow Training Runtime.
+from kubeflow.trainer import *
+
+client = TrainerClient()
+runtime = client.get_runtime("torchtune-llama3.2-1b")
+print(runtime)
+```
+
+Output:
+
+```sh
+Runtime(name='torchtune-llama3.2-1b', trainer=Trainer(trainer_type=<TrainerType.BUILTIN_TRAINER: 'BuiltinTrainer'>, framework=<Framework.TORCHTUNE: 'torchtune'>, entrypoint=['tune', 'run'], accelerator='Unknown', accelerator_count='2'), pretrained_model=None)
+```
+
+## Use TorchTune LLM Trainer in BuiltinTrainer
+
+The guide below shows how to fine-tune Llama-3.2-1B-Instruct with alpaca dataset by BuiltinTrainer.
+
+### Create PVCs for Models and Datasets
+
+{{% alert title="Note" color="info" %}}
+Currently, we do not support automatically orchestrate the volume claim.
+
+To follow up with this problem, please refer to [this issue](https://github.com/kubeflow/trainer/issues/2630).
+{{% /alert %}}
+
+We need to manually create PVCs for each models we want to fine-tune. Please note that **the PVC name must be equal to the ClusterTrainingRuntime name**. In this example, it's `torchtune-llama3.2-1b`.
+
+```Python
+# Create a PersistentVolumeClaim for the TorchTune Llama 3.2 1B model.
+client.core_api.create_namespaced_persistent_volume_claim(
+  body=client.V1PersistentVolumeClaim(
+    api_version="v1",
+    kind="PersistentVolumeClaim",
+    metadata=client.V1ObjectMeta(name="torchtune-llama3.2-1b"),
+    spec=client.V1PersistentVolumeClaimSpec(
+      access_modes=["ReadWriteOnce"],
+      resources=client.V1ResourceRequirements(
+        requests={"storage": "20Gi"}
+      ),
+    ),
+  ),
+)
+```
+
+### Use TorchTune LLM Trainer with train() API
+
+You need to provide the following parameters to use TorchTune LLM Trainer with `train()` API:
+
+- Reference to existing Runtimes.
+- Initializer parameters.
+- Trainer configuration.
+
+Kubeflow TrainJob will train the model in the referenced (Cluster)TrainingRuntime.
```

Comment:
We don't support TrainingRuntime in SDK today, let's just call it: in the referenced TorchTune runtime, like we do for PyTorch: https://www.kubeflow.org/docs/components/trainer/user-guides/pytorch/#get-pytorch-runtime-packages